src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+server.port=8080
+
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2
+
+spring.datasource.url=jdbc:h2:file:~/products
+spring.datasource.username=sa
+spring.datasource.password
+spring.datasource.driver-class-name=org.h2.Driver
```

Comment:
why are you using a database here ?

Reply:
Because it has the service and database implemented without the queue.

Reply:
Is it something you are planning to change later?

Reply:
No, I plan to do it everything within the same project. I might change the database if I have problems further, otherwise I don't I am going to change it.